src/main/java/org/ayty/hatcher/api/v1/course/exception/ObjectNotFoundException.java

```diff
@@ -0,0 +1,18 @@
+package org.ayty.hatcher.api.v1.course.exception;
+
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.BAD_REQUEST)
```

Comment:
Para exceptions de objeto não encontardo, deve retornar HttpStatus.NOT_FOUND. Uma mensagem de erro deve ser retornada, Ex: Course with id 8 not found.

Reply:
O famoso 404 not found